<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArgumentDialog</class>
 <widget class="QDialog" name="ArgumentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxArgument">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Argument</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBoxArguments"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxInputType">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Input Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonFileInput">
        <property name="text">
         <string>File</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButtonTextInput">
        <property name="text">
         <string>Text</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxInputFile">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Input File</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="lineEditFileInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBrowse">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBoxTextInput">
      <property name="title">
       <string>Input Text</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEditInput">
         <property name="autoFormatting">
          <set>QTextEdit::AutoAll</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBoxTextInputValidationMessages">
      <property name="title">
       <string>Validation Messages</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="textEditInputValidation">
         <property name="autoFormatting">
          <set>QTextEdit::AutoAll</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout3">
       <item>
        <widget class="QPushButton" name="pushButtonInitialize">
         <property name="text">
          <string>Initialize Component</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonFileInput</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBoxInputFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonFileInput</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBoxTextInput</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonTextInput</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBoxTextInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonTextInput</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBoxInputFile</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
